projects/go-lib/src/lib/components/go-toast/go-toast.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <div
     class="go-toast__container"
-    [ngClass]="{ 'go-toast__container--max-height': enableMaxHeight }">
+    [ngClass]=" enableMaxHeight?'go-toast__container--max-height':'go-toast__container' ">
```

Comment:
Remove the spaces at the beginning and end of the [ngClass] string value.

Reply:
Spacing has been removed.

Reply:
@adhiyan-tangoe change this
`[ngClass]=" enableMaxHeight?'go-toast__container--max-height':'go-toast__container' "`
to this
`[ngClass]="enableMaxHeight ? 'go-toast__container--max-height' : 'go-toast__container'"`

I meant spaces at the beginning and end of the whole string, not spaces within the string.